src/components/notifications/DingDing.vue

```diff
@@ -6,11 +6,14 @@
         <label for="secretKey" class="form-label">{{ $t("SecretKey") }}<span style="color: red;"><sup>*</sup></span></label>
         <input id="secretKey" v-model="$parent.notification.secretKey" type="text" class="form-control" required>
 
+        <label for="isAtAll" class="form-label">{{ $t("isAtAll") }}<span style="color: red;"><sup>*</sup></span></label>
```

Comment:
Please change this to a selection box with the options `Don't mention people`, `Notify <code>@everyone</code>` (see https://vue-i18n.intlify.dev/guide/advanced/component.html)
With a selection box, implemneting things like `@userId` is possible (=> "future-proof")